.github/workflows/cmake_test.yml

```diff
@@ -78,6 +79,20 @@ jobs:
             -D FUZZTEST_BUILD_TESTING=on \
           && cmake --build build -j $(nproc) \
           && ctest --test-dir build -j $(nproc) --output-on-failure
+      - name: Run all tests in default mode with gcc
+        if: matrix.mode == 'default'
+        run: |
+          CC=gcc CXX=g++ cmake \
+            -S . \
+            -B build \
+            -G Ninja \
+            -D CMAKE_C_COMPILER_LAUNCHER=ccache \
+            -D CMAKE_CXX_COMPILER_LAUNCHER=ccache \
+            -D CMAKE_BUILD_TYPE=RelWithDebug \
+            -D FUZZTEST_BUILD_TESTING=on \
+          && cmake --build build -j $(nproc)
+#          TODO: Rewrite helper `PrintValue` to obtain same output under clang and gcc
```

Comment:
@aleksisch, just wanted to check if you are planning to do this.

Reply:
> @aleksisch, just wanted to check if you are planning to do this.

Yes, I'll investigate it